back/app/controllers/web_api/v1/idea_feed/ideas_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module WebApi
+  module V1
+    module IdeaFeed
+      class IdeasController < ::ApplicationController
+        skip_before_action :authenticate_user
+        before_action :set_phase, only: [:index]
+
+        def index
+          ideas = policy_scope(@phase.ideas.published)
+          ideas = paginate(ideas)
```

Comment:
This will change in a follow-up PR, we won't do pagination on this endpoint, instead just add a parameter to limit the number of returned items but always only get the first page, so without the ability to specify the offset.